landlab/plot/landslides/plot_landslides.py

```diff
@@ -0,0 +1,210 @@
+                value=name, value1=field_min, value2=field_max
+            )
+        )
+        print_value = print(
+            "All {value} default value initialized for each node:".format(value=name)
+        )
+        field = ls_prob.grid["node"][name]
+        print(field)
+
+        print("")
+
+
+def print_list_nodevalues(ls_prob, values_list):
+    """Print out the input range and resulting node values for 
+    all landslide component values on the node.
+    ----------
+    Input: Landlab landslide model instance  """
+
+    for name in values_list:
+        field_min = ls_prob.grid["node"][name].min()
+        field_max = ls_prob.grid["node"][name].max()
+
+        print_minmax = print(
+            "{value} (Min, Max) range of ({value1},{value2}) values on node:".format(
+                value=name, value1=field_min, value2=field_max
+            )
+        )
+        print_value = print(
+            "All {value} default value initialized for each node:".format(value=name)
+        )
+        field = ls_prob.grid["node"][name]
+        print(field)
+
+        print("")
+
+
+def plot_landslide_4variables(
+    littleplots,
+    inputgrid1,
+    inputgrid2,
+    inputgrid3,
+    inputgrid4,
+    variable,
+    maxval,
+    plot_subtitle,
+    cmap,
+    scalelabel,
+):
+    """Return figure with four subplots given landslide variables.
+    Parameters
+    ----------
+    name : littleplots
+    List of subplot matrix locations: row, column, index
+    Examples: 
+    littleplots = [221,222,223,224]
+    littleplots = [411,412,413,414]
+    littleplots = [141,142,143,144]
+    """
+
+    fig = plt.figure("Plot Component Inputs/Outputs")
+
+    ax1 = []
+    ax2 = []
+    ax3 = []
+    ax4 = []
+    for (
+        eachgrid,
+        plotvar,
+        maxval,
+        plot_name,
+        cmap,
+        cbar_name,
+        littleplots,
+        axeslist,
+    ) in zip(
+        [inputgrid1, inputgrid2, inputgrid3, inputgrid4],
+        [
+            inputgrid1.at_node[variable[0]],
+            inputgrid2.at_node[variable[1]],
+            inputgrid3.at_node[variable[2]],
+            inputgrid4.at_node[variable[3]],
+        ],
+        maxval,
+        plot_subtitle,
+        [cmap[0], cmap[1], cmap[2], cmap[3]],
+        [scalelabel[0], scalelabel[1], scalelabel[2], scalelabel[3]],
+        littleplots,
+        [ax1, ax2, ax3, ax4],
+    ):
+
+        axeslist = fig.add_subplot(littleplots)
+        axeslist.xaxis.set_visible(False)
+
+        imshow_grid(
+            eachgrid,
+            plotvar,
+            plot_name=plot_name,
+            cmap=cmap,
+            grid_units=("coordinates", "coordinates"),
+            shrink=0.75,
+            limits=(0, maxval),
+            var_name=cbar_name,
+        )
+
+    fig.set_size_inches(10, 10)
+    return None
+
+
+def scenario_unit_explorer(rw, a, T, theta, hs):
+    print("Given relative wetness = {value}".format(value=rw))
```

Comment:
This doesn't seem appropriate for the landlab.plot submodule. It also needs documentation.